meshtastic/tests/test_node_runtime_transport.py

```
@@ -0,0 +1,1296 @@
+            ch.index = i
+            if i == 0:
+                ch.role = channel_pb2.Channel.Role.PRIMARY
+            elif i == 1:
+                ch.role = channel_pb2.Channel.Role.SECONDARY
+            else:
+                ch.role = channel_pb2.Channel.Role.DISABLED
+            channels.append(ch)
+
+        mock_local_node.channels = channels
+
+        def make_cache_none(*_args: Any, **_kwargs: Any) -> None:
```

Ruff (0.15.6)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `*_args`

(ANN401)